Dockerfile

```diff
@@ -3,23 +3,57 @@
 FROM ubuntu:16.04
 MAINTAINER Michael Stucki <michael@stucki.io>
 
-ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Thanks for reviewing and merging so quick.

Setting `DEBIAN_FRONTEND` to noninteractive via `ENV` is discouraged. That's why I changed it. See [on this docker thread](https://github.com/moby/moby/issues/4032#issuecomment-34597177).

Therefore exporting `DEBIAN_FRONTEND` solely on shell during docker build process is the right choice.